crates/executor/src/executor/mod.rs

```diff
@@ -239,8 +241,10 @@ where
                     .flatten(),
             );
             // Ensure the receipt is not an EIP-7702 receipt.
-            if matches!(receipt, OpReceiptEnvelope::Eip7702(_)) {
-                panic!("EIP-7702 receipts are not supported by the fault proof program");
+            if matches!(receipt, OpReceiptEnvelope::Eip7702(_)) && !is_isthmus {
+                panic!(
+                    "EIP-7702 receipts are not supported by the fault proof program before Isthmus"
```

Comment:
let's not panic here, and rather return/log an error? otherwise this is a possibly a dos vector if we let this slip through somewhere else. usually we avoid panics in client software, since liveness is one of they key properties we're after for network security. assuming it's the same for provers.